shotover-proxy/tests/test-configs/opensearch-passthrough/docker-compose.yaml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+  opensearch-node1:
+    image: opensearchproject/opensearch:latest
```

Comment:
we should specify a complete tag instead of latest, otherwise our tests will suddenly break one day.